modules/frontend/journal/JournalFrontendModule.php

```diff
@@ -87,20 +95,22 @@ private function renderRecentJournalEntry($mJournalId) {
 		return $oTemplate;
 	}
 
-	private function renderRecentJournalCommentTeaser($mJournalId) {
-		$oJournalComment = JournalCommentQuery::create()->excludeUnverified()->joinJournalEntry()->useQuery('JournalEntry')->filterByJournalId($mJournalId)->excludeDraft()->endUse()->mostRecentFirst()->findOne();
+	private function renderRecentJournalCommentTeasers($mJournalId, $iLimit) {
+		$aJournalComments = JournalCommentQuery::create()->excludeUnverified()->joinJournalEntry()->useQuery('JournalEntry')->filterByJournalId($mJournalId)->excludeDraft()->endUse()->mostRecentFirst()->limit($iLimit)->find();
 
-		if($oJournalComment === null) {
+		if($aJournalComments == null) {
```

Comment:
I’m not sure this will ever be `null` anymore. Use:

```suggestion
		if(empty($aJournalComments)) {
```

or drop the check entirely since `foreach` over an empty does not throw an error.